e2e/__tests__/fakeTimersLegacy.test.ts

```diff
@@ -40,13 +39,10 @@ describe('requestAnimationFrame', () => {
 
 describe('setImmediate', () => {
   test('fakes setImmediate', () => {
-    // Jasmine runner does not handle unhandled promise rejections that are causing the test to fail in Jest circus
-    const expectedExitCode = isJestJasmineRun() ? 0 : 1;
-
```

Comment:
would be cool to have a version of this test that runs with the flag active and then exits with `1`